sonar-slang-plugin/src/test/java/com/sonarsource/slang/kotlin/KotlinConverterTest.java

```diff
@@ -617,6 +619,62 @@ public void testAssignments() {
       .isEquivalentTo(slangStatements("x = 3; x -= y + 3;"));
   }
 
+  @Test
+  public void testBreakContinue() {
+    LoopTree tree = (LoopTree) kotlinStatement("while(true)\nbreak;");
+    assertThat(tree.body()).isInstanceOf(JumpTree.class);
+    JumpTree jumpTree = (JumpTree) tree.body();
+    assertThat(jumpTree.kind()).isEqualTo(JumpTree.JumpKind.BREAK);
+    assertThat(jumpTree.keyword().text()).isEqualTo("break");
+    assertThat(jumpTree.label()).isNull();
+
+    tree = (LoopTree) kotlinStatement("while(true)\ncontinue;");
+    assertThat(tree.body()).isInstanceOf(JumpTree.class);
+    jumpTree = (JumpTree) tree.body();
+    assertThat(jumpTree.kind()).isEqualTo(JumpTree.JumpKind.CONTINUE);
+    assertThat(jumpTree.keyword().text()).isEqualTo("continue");
+    assertThat(jumpTree.label()).isNull();
+
+    assertTrees(kotlinStatements("while(true)\nbreak;"))
+      .isEquivalentTo(slangStatements("while(true)\nbreak;"));
+
+    assertTrees(kotlinStatements("while(true)\ncontinue;"))
+      .isEquivalentTo(slangStatements("while(true)\ncontinue;"));
+
+    assertTrees(kotlinStatements("while(true)\nbreak@foo;"))
+      .isEquivalentTo(slangStatements("while(true)\nbreak foo;"));
+
+    assertTrees(kotlinStatements("while(true)\ncontinue@foo;"))
+      .isEquivalentTo(slangStatements("while(true)\ncontinue foo;"));
+
+  }
+
+  @Test
+  public void testReturn() {
+    Tree tree = kotlinStatement("return 2;");
+    assertThat(tree).isInstanceOf(ReturnTree.class);
+    ReturnTree returnTree = (ReturnTree) tree;
+    assertThat(returnTree.keyword().text()).isEqualTo("return");
+    assertThat(returnTree.body()).isInstanceOf(LiteralTree.class);
+
+    tree = kotlinStatement("return;");
+    assertThat(tree).isInstanceOf(ReturnTree.class);
+    returnTree = (ReturnTree) tree;
+    assertThat(returnTree.keyword().text()).isEqualTo("return");
+    assertThat(returnTree.body()).isNull();
+
+    assertTree(kotlinStatement("return 2;"))
+      .isEquivalentTo(slangStatement("return 2;"));
+
+    assertTree(kotlinStatement("return;"))
+      .isEquivalentTo(slangStatement("return;"));
+
+    assertTree(kotlinStatement("return@foo;"))
```

Comment:
why they are equivalent?

Reply:
Inside `KotlinTreeVisitor` I'm not checking if `KtReturnExpression` has a label, so `return@foo` is mapped to a Slang `ReturnTree` with a `null` body.
I will add a check to consider labels when converting `KtReturnExpression`.